src/main/java/yevhen/bookstore/model/ShoppingCart.java

```diff
@@ -0,0 +1,32 @@
+package yevhen.bookstore.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "shopping_carts")
+public class ShoppingCart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @OneToOne(fetch = FetchType.LAZY, optional = false)
```

Comment:
I would suggest using @MapsId to share the user id for the shopping cart (but in that case you need to remove @GeneratedValue annotation from id field(here in this class)). But its up to you =)

Reply:
thank you for advice, it is good improvement